apps/mobile/src/modules/documents/documents.services.ts

```diff
@@ -1,31 +1,33 @@
 import type { ApiClient } from '../api/api.client';
-import type { CoerceDates } from '../api/api.models';
+import type { CoerceDates, LocalDocument } from '../api/api.models';
 import type { AuthClient } from '../auth/auth.client';
 import type { Document } from './documents.types';
 import * as FileSystem from 'expo-file-system/legacy';
 import { coerceDates } from '../api/api.models';
 
-export function getFormData(pojo: Record<string, string | Blob>): FormData {
+export function getFormData(localDocument: LocalDocument): FormData {
```

Comment:
This was  kinda of a generic function to build a form data from a pojo, maybe keep the signature to a Record and use it as before `getFormData({ file })`

```suggestion
export function getFormData(pojo: Record<string, string | LocalDocument>): FormData {
```